src/RecoveryServices/RecoveryServices/help/Get-AzRecoveryServicesBackupItem.md

```diff
@@ -286,7 +286,7 @@ Workload type of the resource. The acceptable values for this parameter are:
 Type: Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.WorkloadType
 Parameter Sets: GetItemsForContainer, GetItemsForVault
 Aliases:
-Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder, SAPHanaDatabase
+Accepted values: AzureVM, AzureFiles, MSSQL, FileFolder
```

Comment:
`SAPHanaDatabase` is still accept. Please submit a issue if you meet problem with `SAPHanaDatabase`.